app/connectors/connectors/sql/src/main/resources/META-INF/syndesis/connector/sql.json

```diff
@@ -224,7 +224,7 @@
                 "secret": false,
                 "componentProperty": false,
                 "defaultValue": "",
-                "description": "StoredProcedure template to perform."
+                "description": "Stored procedure template to perform."
```

Comment:
There was another instance of this, line 284, that instance was "Stored Procedure template to perform." but it should have a lowercase for "procedure". Did that one get updated?